precompile/contract/interfaces.go

```diff
@@ -73,6 +68,7 @@ type Configurator interface {
 		chainConfig precompileconfig.ChainConfig,
 		precompileconfig precompileconfig.Config,
 		state StateDB,
-		blockContext ConfigurationBlockContext,
+		blockNumber *big.Int,
+		blockTime uint64,
```

Comment:
Nice! Simple types >> interface that returns stuff

Reply:
Also using an interface might suggest it could return a different block number or timestamp depending when it's called - that's why you would have methods. So just injecting dumb types makes it more obvious it doesn't change 😉